Riot/Modules/Common/Recents/RecentsViewController.m

```diff
@@ -2526,82 +2519,6 @@ - (void)showEmptyViewIfNeeded
 //        contentViewBottomConstraint.active = YES;
 //    }
 //}
-=======
-- (void)showEmptyView:(BOOL)show
```

Comment:
Please keep the code coming from Element iOS and comment it for Tchap if you want.

Reply:
Fixed in the last commit.